### PR TITLE
RWA-137 Setup authorisation for role assignment.

### PR DESCRIPTION
**Pull request by chrisgrimble**
Corrected setup of role-assignment as we had some issue causing unauthorised exceptions

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RWA-137

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: scripts/organisational-role-assignment.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Options:
+##    - username: Role assigned to user in generated token. Default to `ccd-import@fake.hmcts.net`.
+##    - password: Password for user. Default to `London01`. 
+##    - microservice_name: Name of the microservice. Default to `ccd_gw`.
+##
+## Returns a valid IDAM service token for the given microservice.
+
+USERNAME=${1:-ccd-import@fake.hmcts.net}
+PASSWORD=${2:-London01}
+MICROSERVICE="${3:-ccd_gw}"
```
Comment (alexbaeza) [inline scripts/organisational-role-assignment.sh:13]: is the microservice used anywhere ?

File: values/am-role-assignment.yaml.gotmpl [hunk 2]
```diff
@@ -20,17 +20,17 @@ app:
...
       - key: "IDAM_USER_URL"
         value: http://sidam-api
       - key: "IDAM_S2S_URL"
         value: http://service-auth-provider-api
       - key: "ROLE_ASSIGNMENT_IDAM_CLIENT_ID"
-        value: am_docker
+        value: ccd_gateway
       - key: "ROLE_ASSIGNMENT_IDAM_CLIENT_SECRET"
-        value: am_docker_secret
+        value: OOOOOOOOOOOOOOOO
```
Comment (alexbaeza) [inline values/am-role-assignment.yaml.gotmpl:33]:
```suggestion
        value: {{ requiredEnv "OAUTH2_CLIENT_CCD_GATEWAY" }}
```

File: scripts/test_script.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (alexbaeza) [inline scripts/test_script.sh:1]: Shall we remove this one ?

File: scripts/organisational-role-assignment.sh (hunk 1 above)
Comment (chrisgrimble) [reply scripts/organisational-role-assignment.sh:13]: Ah thats a bug should have been passed to the idam0service-token.sh script

File: scripts/test_script.sh (hunk 3 above)
Comment (chrisgrimble) [reply scripts/test_script.sh:1]: yep

**Review by alexbaeza — APPROVED** [review]

**Pull request merged**